src/VisualizationTests/HelixImageComparisonTests.cs

```diff
@@ -68,9 +70,10 @@ private static System.Drawing.Bitmap BitmapFromSource(BitmapSource bitmapsource)
             return bitmap;
         }
 
-        private string GenerateTestDataPathFromTest(string testname)
+        private string GenerateTestDataPathFromTest(string testname,bool debug = false)
```

Comment:
Could we rename this to `GenerateImagePathFromTest` unless we are generating any other "data" than just images?

Reply:
yes can do.